docs/products/kafka/concepts/topic-catalog-overview.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Aiven for Apache Kafka® topic catalog
+sidebar_label: Topic catalog
+limited: true
+---
+
+The Aiven for Apache Kafka® topic catalog provides a centralized interface within the Aiven console to view and manage Apache Kafka topics across different projects and services.
+This unified view consolidates all topics into one accessible location, streamlining
+the management of your Aiven for Apache Kafka infrastructure.
+
+## Key features and benefits
+
+- **Centralized management**: Access all Apache Kafka topics across your organization
+  and projects from a single interface.
+- **Search topics**: Find specific Apache Kafka topics using the search bar.
+- **Topic information at a glance**: View topic names, associated services, projects,
+  and owners in either table view or tile view.
```

Comment:
tile view is currently only available when governance is enabled do we want to change that so it is available to anyone using the topic catalog?

Reply:
I can see both views (Table and Tile) on the dev-sandbox without icons. However, it would be beneficial to have these views available regardless of whether governance is enabled.